### DISCUSSION TITLE
BackdropView for Android and iOS (And maybe other platforms)

### DISCUSSION THREAD

**Discussion by mabroukmahdhi** [CommunityToolkit/Maui#961] [New Feature Discussions]
I would like to implement a Backdrop View/Control for iOS and Android. 

Something like that:
![image](https://user-images.githubusercontent.com/16063715/216724936-566e376e-3fcf-4139-8a4b-89b4578bc3a1.png)

**Comment by VladislavAntonyuk** [comment]
It looks similar to the BottomSheet: https://vladislavantonyuk.github.io/articles/Creating-a-bottom-sheet-using-.NET-MAUI
Does it cover your needs?

**Reply by mabroukmahdhi** [reply]
I guess yeah! Thanks! I will check this

**Comment by CliffAgius** [comment]
Closed as per the Community Stand-up discussion here: https://youtu.be/N9wMcBP4jtg?t=2889
